.github/scripts/terraform_tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o pipefail
+
+ERROR() { echo "::error::$*"; }
+INFO() { echo "::info::$*"; }
+DEBUG() { test -z "$VERBOSE" || echo "::debug::$*"; }
+
+OBSERVE_DOMAIN="${OBSERVE_DOMAIN:-observe-staging.com}"
```

Comment:
ditch this, it won't be a sane default for external users. It's better for our usecase to be explicit anyway, and since inherit secrets there's no duplication to worry about.

Reply:
👍